sdk/sdk-view/src/test/java/mil/emp3/api/CameraLookAtTest.java

```diff
@@ -102,37 +118,56 @@ public void cameraTests() {
 
     @Test
     public void cameraSettingsApplyTest() {
+        final boolean[] resultFound = {false};
+        final boolean[] testPassed = {false};
+
         String userContext = "Camera test AFTER";
         camera.setAltitude(10000.0);
+        camera.setAltitudeMode(IGeoAltitudeMode.AltitudeMode.CLAMP_TO_GROUND);
         camera.setLatitude(global.LATITUDE_MAXIMUM);
         camera.setLongitude(global.LONGITUDE_MAXIMUM);
         camera.setRoll(global.CAMERA_ROLL_LEVEL);
         camera.setHeading(global.HEADING_MAXIMUM);
         camera.setTilt(global.CAMERA_TILT_TO_SKY);
-        mapInstance.applyCameraChange(camera, false, userContext);
-//        mapInstance.setCamera(camera, false,userContext);
+//        mapInstance.applyCameraChange(camera, false, userContext);
+        mapInstance.setCamera(camera, false,userContext);
```

Comment:
@rsupnekar It seems you added this test a month ago.  Do you know why you used applyCameraChange instead of setCamera?  From what I can tell by using applyCameraChange it prevents the code from generating an eventListener